operator_ui/src/api/index.ts

```diff
@@ -1,4 +1,10 @@
-import * as v2 from './v2'
+import { Api } from '@chainlink/json-api-client'
+import { Sessions } from './sessions'
+import { V2 } from './v2'
 
-export * from './sessions'
-export { v2 }
+const api = new Api({
+  base: process.env.CHAINLINK_URL,
```

Comment:
How do you feel about renaming this to `CHAINLINK_BASEURL` to keep it consistent?

There are a few integration scripts that would need to be updated.